### DISCUSSION TITLE
federation-graphql-java-support

### DISCUSSION THREAD

**Discussion by deweyjose** [Netflix/dgs-framework#1520] [Q&A]
Hi team,

Our team needs to use [v2.3.1](https://github.com/apollographql/federation-jvm/releases/tag/v2.3.1) of apollo federation `federation-graphql-java-support`. 

It looks like DGS has a dependency on 2.1.1 and 2.3.1 of `federation-graphql-java-support` seems to be compatible, I don't see any breaking changes in apollo's subsequent releases.

Any thoughts on your end on updating this in DGS proper and on our use of 2.3.1 ahead of that?

**Comment by srinivasankavitha** [comment]
Thanks for bringing this to our attention. Our master just got updated to v3.0.0 as part of this [PR](https://github.com/Netflix/dgs-framework/pull/1523) and we will be releasing this early next week
